storage/ndb/src/kernel/vm/mt.cpp

```diff
@@ -2787,8 +2803,7 @@ thr_send_threads::assign_threads_to_assist_send_threads()
     thr_data *selfptr = &rep->m_thread[thr_no];
     selfptr->m_nosend = conf.do_get_nosend(selfptr->m_instance_list,
                                            selfptr->m_instance_count);
-    if (is_recover_thread(thr_no) ||
-        selfptr->m_nosend == 1)
+    if (selfptr->m_nosend == 1)
```

Comment:
We first round robin the sending of the LDM threads and next we round robin on all the other thread types, this is to avoid several LDM threads sharing the same send thread

Reply:
I see. I think each type of thread is continuous in rep->m_thread[], so it won't be a problem to use only one for-loop.

Reply:
Possibly true, but not sure I want to rely on that, sort of hidden logic.